scripts/partition_manager.py

```diff
@@ -1100,19 +1101,19 @@ def test():
     # Test a single partition with alignment where the address is smaller than the alignment value.
     td = {'without_alignment': {'placement': {'before': 'with_alignment'}, 'size': 100},
           'with_alignment': {'placement': {'before': 'app', 'align': {'start': 200}}, 'size': 100},
-          'app': {}}
-    s, sub_partitions = resolve(td)
-    set_addresses_and_align(td, sub_partitions, s, 1000)
+          'app': {'region': 'flash_primary',}}
```

Comment:
Why this change? I see it's done many places. If it's an unrelated fix, maybe give it a separate commit.

Reply:
Its in the wrong commit, I moved it to the correct commit now, should make sense.